locale/it.json

```diff
@@ -38,7 +38,7 @@
   "question.2key.remove": "Desideri rimuovere %1 dall'elenco dei siti attendibili?",
   "warning.title.oh_no": "Oh no, non ha funzionato!",
   "warn.ssl.install": "È necessario installare il certificato SSL radice di Fortify. Per eseguire questa operazione potrebbe essere necessaria la password di amministratore.",
-  "warn.ssl.renew": "SSL certificate requires renew. Please run the Fortify installer to renew a certificate.",
+  "warn.ssl.renew": " Il certificato SSL di Fortify deve essere rinnovato. Esegui l'installer di Fortify per rinnovare il certificato.",
```

Comment:
Please remove the empty space before ` Il certificato SSL...`.

Reply:
Resolved the typo